### PR TITLE
chore(deps) bump excalidraw version

### PR DESCRIPTION
**Pull request by mihhu**

### CODE REVIEW

**Review by saghul — APPROVED** [review]

**Comment by mihhu** [comment]
```
ERROR in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (5 MiB). This can impact web performance. 
33Entrypoints: 
34  app.bundle (12 MiB) 
35      app.bundle.min.js
```

Shall we ditch the source map in this case, @saghul?

**Comment by saghul** [comment]
The source map should be a separate file, just like other bundles.

**Comment by mihhu** [comment]
It is, then I'm not sure what causes the huge bump in the app.bundle entrypoint size.. Can I just update the performance values?

**Comment by saghul** [comment]
No way we are almost tripling the size.

Check with the webpack bundle analyzer. It shows a nice visual which would reveal where the size increase comes from.

File: package.json [hunk 1]
```diff
@@ -179,7 +179,7 @@
     "typescript": "5.0.4",
     "unorm": "1.6.0",
     "webpack": "5.76.0",
-    "webpack-bundle-analyzer": "4.4.2",
+    "webpack-bundle-analyzer": "^4.4.2",
```
Comment (saghul) [inline package.json:182]: Please drop the caret

**Comment by saghul** [comment]
Is the bundle size issue fixed? And we get source maps? Exciting!

**Comment by mihhu** [comment]
> Is the bundle size issue fixed? And we get source maps? Exciting!

No.. I just thought to get the fix in to catch the release asap. I'll get on the bundle size too

**Review by saghul — APPROVED** [review]

**Pull request merged**